src/main/webapp/WEB-INF/spring/webmvc-config.xml

```diff
@@ -25,61 +25,25 @@
     <!-- Handles HTTP GET requests for /resources/** by efficiently serving up static resources -->
     <mvc:resources location="/, classpath:/META-INF/web-resources/" mapping="/resources/**"/>
     
-    <mvc:resources mapping="/public/docs/**" location="public/docs/" />
```

Comment:
This was all stuff used in roo, we no longer have these pages and this is no longer relevant to this application.